Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mitmproxy/mitmproxy:latest
```

Comment:
Might wanna pin this to a specific version, so it doesn't keep moving under our feet.